### PR TITLE
Full Audit: Working with Checks

### PR DESCRIPTION
**Pull request by DonNateR**
Discovered these while writing the new article about custom dashboards.

### CODE REVIEW

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-426fd760e0/how-to/)

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-0971c4a5f7/how-to/)

**Comment by kmsholcomb** [comment]
Hi @DonNateR. Edited are attached. Please let me know if you have any questions. Thanks!

[Understand and work with checks_editsKH.docx](https://github.com/rackerlabs/rackspace-how-to/files/1218945/Understand.and.work.with.checks_editsKH.docx)
[Understand and work with checks_editsKH.pdf](https://github.com/rackerlabs/rackspace-how-to/files/1218946/Understand.and.work.with.checks_editsKH.pdf)

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-e98d0818cb/how-to/)

File: content/rackspace-intelligence/working-with-checks.md [hunk 1]
```diff
@@ -1,40 +1,29 @@
...
-Checks are of the *agent* type if they monitor an entity's resource utilization.
-Agent-type
-checks are performed by the monitoring agent running on a server.
-For example, a **Memory** check can trigger an alarm if it detects
-memory usage above a specified percentage. Agent checks require the
-monitoring agent to be installed on the server. For a list of
-available agent check types, see
-[Agent check types](https://developer.rackspace.com/docs/cloud-monitoring/v1/developer-guide/#agent-check-types).
-For information on how to install the monitoring agent, see [Install and configure](https://developer.rackspace.com/docs/cloud-monitoring/v1/developer-guide/#document-install-configure).
+*Agent check*s monitor an entity's resource utilization. Agent checks are performed by the monitoring agent that is running on a server. For example, a Memory check monitors a server's memory use and can trigger an alarm if it detects memory use above a specified percentage.
```
Comment (stephamon) [inline content/rackspace-intelligence/working-with-checks.md]: The asterisk at the end of *agent checks* should be after the 's'

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-1a715c4b5a/how-to/).

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-5786ae3d86/how-to/)

**Comment by stephamon** [comment]
👍

**Pull request merged**